tests/test_freshness_date_parser.py

```diff
@@ -353,6 +368,8 @@ def test_relative_past_dates_with_time_as_period(self, date_string, ago, period)
         param('1 वर्ष, 8 महीने, 2 सप्ताह', ago={'years': 1, 'months': 8, 'weeks': 2}, period='week'),
         param('1 वर्ष 7 महीने', ago={'years': 1, 'months': 7}, period='month'),
         param('आज', ago={'days': 0}, period='day'),
+        param('1 दशक', ago={'years': 10}, period='year'),
+        param('1 दशक पहले', ago={'years': 10}, period='year'),
```

Comment:
I have added a few suggestions below to improve the Hindi test cases. Please have a look.

Reply:
@gutsytechster Done this change, Please Look into it 😊.

Reply:
I believe these test cases can be added here as well
- 1 दशक पूर्व
- दो दशक पहले
- 10 दशकों पहले